src/core/components/live-response.jsx

```diff
@@ -63,9 +64,7 @@ export default class LiveResponse extends React.Component {
         { curlRequest && <Curl request={ curlRequest }/> }
         { url && <div>
             <h4>Request URL</h4>
```

Comment:
Please add "<h4> Request Url </h4>" to the RequestUrl component

Reply:
done